src/en/mayotune/src/eu/kanade/tachiyomi/extension/en/mayotune/MayoTune.kt

```diff
@@ -0,0 +1,129 @@
+package eu.kanade.tachiyomi.extension.en.mayotune
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.parseAs
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+
+class MayoTune() : HttpSource() {
+    override val name: String = "MayoTune"
+    override val baseUrl: String = "https://mayotune.xyz"
+    override val lang: String = "en"
+    override val versionId: Int = 1
+
+    private val source = SManga.create().apply {
+        title = "Mayonaka Heart Tune"
+        url = baseUrl
```

Comment:
This should also be relative, as with chapter URLs. IIRC it makes it easier to migrate to a different domain for existing users.
```suggestion
        url = "/"
```

Needs other changes.

Reply:
I tried to figure this one out before. 
Apparently without override `mangaDetailsRequest`, relative URLs gets resolved into `baseUrl` anyways, so the address becomes `<baseUrl><baseUrl>`. I can't think of any other ways to do it.

Reply:
Nvm just changing the URL was enough, should be good now